services/subscriber/service.go

```diff
@@ -285,7 +293,8 @@ func (s *Service) updateSubs(wg *sync.WaitGroup) error {
 				}
 				sub, err := s.createSubscription(se, si.Mode, si.Destinations)
 				if err != nil {
-					return err
+					s.Logger.Printf("Subscription creation failed for: %s", si.Name)
+					continue
```

Comment:
Now that no error is returned we should change the return of the updateSubs method to return nothing, since there is no longer any case where an error is returned.

f we run into this error it means that we have a configured subscription but it is not running. Since there is no way to query the status of the subscriptions we should add a statistic for this error so that it is easier to detect. That way a user can setup an alert on the statistic to know if/when subscriptions are failing.